### DISCUSSION TITLE
【Q】Macを買ったほうが

### DISCUSSION THREAD

**Discussion by Terralien-jp** [Hackathon5-TeamD/evolution-quest#62] [Polls]
どうやらWindows環境は私だけのようで前回に引き続きCRLF改行コード問題が発生しました。
解決策を見つけたものの、手っ取り早くMac買ったほうが早いのではないかと思っています。

**Comment by kobakichi** [comment]
寺さんお金と猫たくさん持ってそうなので、Mac購入に一票です。

(冗談です)
